nginx-agent-docker/Dockerfile

```
@@ -0,0 +1,59 @@
+FROM debian:bullseye-slim
+
+ARG NMS_URL
+ARG DEVPORTAL=false
+ARG NAP_WAF=false
+
+# Initial packages setup
+RUN	apt-get -y update \
+	&& apt-get -y install -y apt-transport-https lsb-release ca-certificates wget gnupg2 wget curl \
```

```suggestion
	&& apt-get -y install -y apt-transport-https lsb-release ca-certificates wget gnupg2 curl \
```
you have `wget` twice. Also do you need both `curl` and `wget`?